CHANGELOG.md

```diff
@@ -1,10 +1,18 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Disable `enableUserInteractionBreadcrumbs` on Android when `enableAutoNativeBreadcrumbs` is disabled ([#1131](https://github.com/getsentry/sentry-dart/pull/1131))
+
 ## 6.15.1
 
-### Various fixes & improvements
+### Dependencies
 
-- chore(deps): update Flutter SDK (metrics) to v3.3.8 (#1121) by @github-actions
+- Bump Cocoa SDK from v7.30.1 to v7.30.2 ([#1113](https://github.com/getsentry/sentry-dart/pull/1113))
+  - [changelog](https://github.com/getsentry/sentry-cocoa/blob/master/CHANGELOG.md#7302)
+  - [diff](https://github.com/getsentry/sentry-cocoa/compare/7.30.1...7.30.2)
```

Comment:
@brustolin you have changed the wrong entry for the changelog, so fixing it here on the fly.